STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -903,6 +909,7 @@ extension to the CSR template.
 [[RFC Editor: please replace XXXX below by the RFC number.]]
 
 ## New Fields in the "meta" Object within a Directory Object
+{: #capability-metadata}
```

Comment:
I think this should point to the "Negotiating an Unauthenticated GET" section rather than the IANA table.

BTW, One pretty cool thing is we could add the anchor to the right paragraph, e.g.:
```
[...]
differ in terms of their position in the directory meta and order objects:
rather than being wrapped in an auto-renewal sub-object they are located at the
top-level.

{: #capability-metadata}
A server states its availability to grant unauthenticated access to a client's
Order certificate by setting the `allow-certificate-get` attribute to `true` in
the `meta` field inside the directory object:
```
which the xml2rfc machinery automatically translates into:
```
3.  CA Behavior

   Although most of this document, and in particular Section 2 is
   focused on the protocol between the NDC and to IdO, the protocol does
   affect the ACME server running in the CA.  A CA that wishes to
   support certificate delegation MUST also support unauthenticated
   certificate fetching, which it declares using "allow-certificate-get"
   (Section 2.3.5, Paragraph 3).
```

Reply:
If you agree I could push my local branch which has this change in (plus a rebase on top of main).

Reply:
Yes please.

Reply:
sure, done in 623f2c8